src/Roslyn.Suppressions/Roslyn.Suppressions/InProcess/RoslynSettingsFileSynchronizer.cs

```diff
@@ -114,18 +122,18 @@ public async Task UpdateFileStorageAsync()
                 };
                 roslynSettingsFileStorage.Update(settings, solutionNameWithoutExtension);
             }
-            else
```

Comment:
It could, so I decided to revert the commit [470f9594](https://github.com/SonarSource/sonarlint-visualstudio/pull/5894/commits/470f9594e086a28afc0d9eb78c71aa00468daed1)